rskj-core/src/test/java/co/rsk/peg/BridgeSerializationUtilsTest.java

```diff
@@ -664,22 +667,20 @@ void serializeAndDeserializeFederationOnlyBtcKeysWithRealRLP() {
         };
 
         // Only actual keys serialized are BTC keys, so deserialization will fill RSK and MST keys with those
-        Federation federation = FederationFactory.buildStandardMultiSigFederation(
-            FederationTestUtils.getFederationMembersWithKeys(Arrays.asList(
-                BtcECKey.fromPublicOnly(publicKeyBytes[0]),
-                BtcECKey.fromPublicOnly(publicKeyBytes[1]),
-                BtcECKey.fromPublicOnly(publicKeyBytes[2]),
-                BtcECKey.fromPublicOnly(publicKeyBytes[3]),
-                BtcECKey.fromPublicOnly(publicKeyBytes[4]),
-                BtcECKey.fromPublicOnly(publicKeyBytes[5])
-            )),
-            Instant.ofEpochMilli(0xabcdef),
-            42L,
-            networkParms
-        );
+        List<FederationMember> members = FederationTestUtils.getFederationMembersWithKeys(Arrays.asList(
+            BtcECKey.fromPublicOnly(publicKeyBytes[0]),
+            BtcECKey.fromPublicOnly(publicKeyBytes[1]),
+            BtcECKey.fromPublicOnly(publicKeyBytes[2]),
+            BtcECKey.fromPublicOnly(publicKeyBytes[3]),
+            BtcECKey.fromPublicOnly(publicKeyBytes[4]),
+            BtcECKey.fromPublicOnly(publicKeyBytes[5])
+        ));
+        FederationArgs federationArgs = new FederationArgs(members, Instant.ofEpochMilli(0xabcdef), 42L,
```

Comment:
i don't think its worth being a class instance value. It's only used in two places in a 1200-line file.
but i agree it could be placed inside a named variable